config/foreman-hiera.conf

```diff
@@ -6,6 +6,7 @@
   - kafo_answers
   - security
   - tuning
+  - mongo-redhat-7
```

Comment:
The redhat should be `%{::osfamily}` and the `7` should be `%{::operatingsystemmajrelease}` so it's only loaded for EL7. You may need to rename the file to `mongo-RedHat-7.yaml`